### PR TITLE
fix page formatting

### PR DESCRIPTION
**Pull request by kbhawkey**
- Fix formatting of task page
- Fix extra end paren

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page previews:
https://deploy-preview-20159--kubernetes-io-master-staging.netlify.com/docs/concepts/cluster-administration/flow-control/
https://deploy-preview-20159--kubernetes-io-master-staging.netlify.com/docs/tasks/setup-konnectivity/setup-konnectivity/

File: content/en/docs/tasks/setup-konnectivity/setup-konnectivity.md [hunk 1]
```diff
@@ -1,30 +1,43 @@
...
 is a reference implementation.
 
-Deploy the Konnectivity server on your master node. The provided yaml assuming
-Kubernetes components are deployed as {{< glossary_tooltip text="static pod"
-term_id="static-pod" >}} in your cluster. If not , you can deploy it as a
-Daemonset to be reliable.
+Deploy the Konnectivity server on your master node. The provided yaml assumes
+that the Kubernetes components are deployed as a {{< glossary_tooltip text="static Pod"
+term_id="static-pod" >}} in your cluster. If not, you can deploy the Konnectivity
+server as a Daemonset.
```
Comment (sftim) [inline content/en/docs/tasks/setup-konnectivity/setup-konnectivity.md:40]: nit:
```suggestion
server as a DaemonSet.
```

Comment (kbhawkey) [reply content/en/docs/tasks/setup-konnectivity/setup-konnectivity.md]: 👍

**Comment by kbhawkey** [comment]
I plan to log an issue about the file location.

**Pull request merged**